libsycl/include/sycl/detail/export.hpp

```diff
@@ -0,0 +1,44 @@
+//==---------------- export.hpp - SYCL standard header file ----------------==//
```

Comment:
@tahonermann do you know if that first line is still required by the upstream polices? I think I've seen something on the Discourse...

Reply:
I'm not sure. The coding standard details for file headers is [here](https://llvm.org/docs/CodingStandards.html#file-headers).

Reply:
updated